<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VentanaHorno</class>
 <widget class="QMainWindow" name="VentanaHorno">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>317</width>
    <height>281</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>317</width>
    <height>281</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>317</width>
    <height>281</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Horno</string>
  </property>
  <property name="windowIcon">
   <iconset resource="imagenes.qrc">
    <normaloff>:/imagenes/Imagenes/iconoHorno.png</normaloff>:/imagenes/Imagenes/iconoHorno.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <widget class="QLabel" name="labelBandejas">
      <property name="text">
       <string>Bandejas</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QCheckBox" name="checkBandeja1">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Bandeja 1</string>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBandeja2">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Bandeja 2</string>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBandeja3">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Bandeja 3</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBandeja4">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Bandeja 4</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBandeja5">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Bandeja 5</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBandeja6">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Bandeja 6</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QPushButton" name="configurarBandeja1">
          <property name="text">
           <string>Configurar</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="configurarBandeja2">
          <property name="text">
           <string>Configurar</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="configurarBandeja3">
          <property name="text">
           <string>Configurar</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="configurarBandeja4">
          <property name="text">
           <string>Configurar</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="configurarBandeja5">
          <property name="text">
           <string>Configurar</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="configurarBandeja6">
          <property name="text">
           <string>Configurar</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="labelHorno">
      <property name="minimumSize">
       <size>
        <width>299</width>
        <height>15</height>
       </size>
      </property>
      <property name="text">
       <string>Horno</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QRadioButton" name="encendido">
        <property name="text">
         <string>Encendido</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="apagado">
        <property name="text">
         <string>Apagado</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="imagenes.qrc"/>
 </resources>
 <connections/>
</ui>
